pro/src/screens/OfferIndividual/StocksEvent/__specs__/StocksEvent.spec.tsx

```diff
@@ -315,4 +315,53 @@ describe('screens:StocksEvent', () => {
       screen.queryByRole('heading', { name: /Stock & Prix/ })
     ).toBeInTheDocument()
   })
+  it('should not display any message when user delete empty stock', async () => {
```

Comment:
Pb de titre, 
(proposition)should display a toaster where user delete a saved stock ?

Reply:
Oui carrément

Reply:
Oui carrément